content/en/agent/guide/network.md

```diff
@@ -322,6 +322,9 @@ Used for Agent services communicating with each other locally within the host on
 8126/tcp
 : Port for the [APM receiver][3]
 
+5012/tcp
```

Comment:
Can you move this up so that it comes after 5002?

Reply:
done!